corrector.rb

```diff
@@ -0,0 +1,11 @@
+class Corrector
+  def correct_name(name)
+    first_letter = name[0].capitalize
+    str = name
+    str[0] = first_letter
+    str[0, 11]
```

Comment:
- [ ] It is requested in the project instructions to make sure that the name has a maximum of 10 characters but it's 11 for you. Let's trim the word,
<img width="713" alt="Capture" src="https://user-images.githubusercontent.com/58504655/135921884-c651f31c-cc2a-49b4-b5f4-8263bc76502f.PNG">

Reply:
:heavy_check_mark: Fixed!